src/main/resources/db/migration/V2__barn.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE Soker (
+    sak           UUID REFERENCES sak (id),
+    navn          VARCHAR,
+    fodselsnummer VARCHAR
+);
+
+CREATE TABLE barn (
+    sak                     UUID REFERENCES sak (id),
+    navn                    VARCHAR,
+    fodselsnummer           VARCHAR,
```

Comment:
Ser dere bare bruker fødselsnummer. Er det bevisst? 
Er det slik at dere ikke kommer til å støtte søkere som har dnr på sikt?

Reply:
For de fleste praktiske formål er det ikke forskjell på fnr/dnr, så jeg pleier bare kalle feltet fødselsnummer.